en/Developers/Build plugins.md

```diff
@@ -0,0 +1,60 @@
+Learn how you can extend Obsidian with more features by building your own plugins and sharing them with the community on the [[Marketplace]].
+
+## Quickstart
+
+The easiest way to get started with Obsidian plugin development is by using the [Obsidian Sample Plugin](https://github.com/obsidianmd/obsidian-sample-plugin).
+
+1. Clone the sample plugin.
+
+   ```bash
+   git clone https://github.com/obsidianmd/obsidian-sample-plugin.git
+   mv obsidian-sample-plugin your-plugin-id
+   cd your-plugin-id
+   ```
+
+1. Install dependencies.
+
+   ```bash
+   # Using NPM
+   npm install
+
+   # Using Yarn
+   yarn install
+   ```
+
+1. Rebuild plugin whenever you edit the code.
+
+   ```bash
+   # Using NPM
+   npm run dev
+
+   # Using Yarn
+   yarn run dev
+   ```
+
+1. Install the plugin to your vault.
+
+   ```bash
+   mkdir $YOUR_VAULT_PATH/.obsidian/plugins/your-plugin-id
+   mv main.js styles.css manifest.json $VAULT_PATH/.obsidian/plugins/your-plugin-id
+   ```
+
+1. Restart Obsidian.
+
+For more help on building plugins for Obsidian, check out the following resources:
```

Comment:
we should add our O_O video playlist here: https://www.youtube.com/watch?v=CtR-d-gyxHg&list=PLIDCb22ZUTBnMCbJa-st4PD5T3Olep078